### ISSUE TITLE
Consent animation embedded on Consent page

### ISSUE THREAD

**Issue by ellemindwave** [research-resource/research_resource#21]
As a user I want to feel informed when it comes to how the consent process works, so that I am prepared and ready, and am aware of everything that is required of me to complete it. 

TTRR team are getting an animation made for the consent process. This may or may not be available during our sprints, and so at the moment isn't a priority

**Comment by Cleop** [comment]
@ellemindwave it would be useful to know which page you imagine this being found on?

**Comment by Cleop** [comment]
@ellemindwave is the animation on what consent is or on how to complete a saliva sample?

**Comment by ellemindwave** [comment]
@Cleop the consent animation will be on the first consent page, where they are directed to after their initial sign up (with just email/phone/password). 

The consent animation will be about the entire consent process, not just giving a saliva sample.

**Comment by Cleop** [comment]
Great, thanks for clarifying @ellemindwave. Perhaps it might be worth updating the user story on this issue. I feel the current one does not reflect the purpose of the consent animation:

'As a user I want to feel informed and ready when it comes to how to complete a saliva sample.'

**Comment by ellemindwave** [comment]
@Cleop Thanks for the pointer - have updated it now to be more relevant

**Comment by Cleop** [comment]
@ellemindwave do you think this animation is likely to be ready before user testing on the 18th April? If not, should the box/space for it be removed from the current live version of the site?

![image](https://cloud.githubusercontent.com/assets/16775804/24913241/71e7acd4-1ec8-11e7-9a36-53605ddb1342.png)

**Comment by ellemindwave** [comment]
@Cleop Yes, that is perhaps a good idea, as it looks a bit odd stuck in there! 
The animation will not be ready by then. 

Please can the box and text above it be hidden for the time being?
